recipes-extended/xen/files/libxl-vwif-support.patch

```diff
@@ -319,17 +319,17 @@ PATCHES
      ("coloft_forwarddev", string),
 --- a/tools/libxl/libxl_types_internal.idl
 +++ b/tools/libxl/libxl_types_internal.idl
-@@ -29,6 +29,7 @@ libxl__device_kind = Enumeration("device
-     (12, "VDISPL"),
+@@ -30,6 +30,7 @@ libxl__device_kind = Enumeration("device
      (13, "VUART"),
      (14, "PVCALLS"),
-+    (15, "VWIF"),
+     (15, "VSND"),
```

Comment:
Ticket number for this change?

Reply:
The related commit can be found here: https://github.com/xen-project/xen/commit/0c798b22a009fd6e3ec388c11c1752e5eaa561aa

No change on our end, just a contextual adjustment to a change in the surrounding code.